### ISSUE TITLE
V1 Patch for Git issues

### ISSUE THREAD

**Issue by Mason-Seeger** [reviewdog/action-brakeman#40]
I forked and created a [branch with a fix for the git issue](https://github.com/Mason-Seeger/action-brakeman/commit/21f953f0944f60a75eda79c3e4d7da4ddd50565e) with the advised fix from https://github.com/reviewdog/action-yamllint/pull/19/files for the issue described in https://github.com/reviewdog/reviewdog/issues/1158. With this being a previous version, I am not sure what the process is but it would be very helpful if this change could be released as v1.7.2

**Comment by LucasArruda** [comment]
@Mason-Seeger nice! 

I added https://github.com/reviewdog/action-brakeman/pull/41 which also fixes it.

Not sure which of the two versions (mine or yours) is the correct one – maybe both.

But while this is not tagged to `v1.7.2` we are using a fork to be able to correctly use this. Hope one of the two will get tagged soon.

**Comment by LucasArruda** [comment]
Also, out of curiosity, why are you using `v1`, @Mason-Seeger?

Somehow, `v2` doesn't work for my client's repo, so we had to resort to `v1`, which kind looks not being so actively built, but nice that we've been able to fix it.

**Comment by Mason-Seeger** [comment]
@LucasArruda I am not sure why v1, probably because there were no issues with it/ we hadn't needed to upgrade. Though I had updated to use V2 with the suggestions found in https://github.com/reviewdog/action-brakeman/issues/38

If it helps here is an example of what my currently working setup looks like for action-brakeman. The biggest thing I saw from upgrading from V1 to V2 was a need to have ruby/setup-ruby run prior to brakeman 

```
brakeman:
    name: Brakeman
    runs-on: ubuntu-latest
    steps:
      - name: Check out code
        uses: actions/checkout@v1

      - uses: ruby/setup-ruby@v1
        with:
          ruby-version: 2.7.2

      - name: brakeman
        uses: reviewdog/action-brakeman@v2
        with:
          brakeman_version: 4.8.2
          github_token: ${{ secrets.github_token }}
          reporter: github-pr-check
          fail_on_error: true
          brakeman_flags: '--no-exit-on-warn --no-exit-on-error'
```

**Comment by Mason-Seeger** [comment]
@LucasArruda I am not sure why v1, probably because there were no issues with it/ we hadn't needed to upgrade. Though I had updated to use V2 with the suggestions found in https://github.com/reviewdog/action-brakeman/issues/38

If it helps here is an example of what my currently working setup looks like for action-brakeman. The biggest thing I saw from upgrading from V1 to V2 was a need to have ruby/setup-ruby run prior to brakeman 

```
brakeman:
    name: Brakeman
    runs-on: ubuntu-latest
    steps:
      - name: Check out code
        uses: actions/checkout@v1

      - uses: ruby/setup-ruby@v1
        with:
          ruby-version: 2.7.2

      - name: brakeman
        uses: reviewdog/action-brakeman@v2
        with:
          brakeman_version: 4.8.2
          github_token: ${{ secrets.github_token }}
          reporter: github-pr-check
          fail_on_error: true
          brakeman_flags: '--no-exit-on-warn --no-exit-on-error'
```

**Comment by LucasArruda** [comment]
Oh, got it. Thanks for the input

I had the same error – `exit code 3`, and the we switched back to `v1` and it worked well.

Tried all that (ruby, etc) and didn't work. But also didn't know about the flags. So might just work if I put them (together with `fail_on_error`) and try using `v2`.

**Comment by LucasArruda** [comment]
@Mason-Seeger I think https://github.com/reviewdog/action-brakeman/pull/42 might fix v2.

**Comment by javierjulio** [comment]
This was fixed by #39 which is included in v2, thank you.
